### ISSUE TITLE
Prepare for changes in NVDA 2023.1

### ISSUE THREAD

**Issue by LeonarddeR** [BabbageCom/unicornNVDA#6]
Braille breaks after pr https://github.com/nvaccess/nvda/pull/14503, as the enabled and displaySize properties on the braille handler are now read only. This is intentional, as NVDA Remote should start to rely on new extension points wherever possible.
        As I'm the author of the breakage, I'll be more than happy to provide a fix-up pr. Lets make NVDA Remote work first and then we should be able to copy the patch pretty easilly.
Cc @FalkoBabbage

**Comment by FalkoBabbage** [comment]
Hi Leonard,
Thanks for the heads up and orry for the late response. I already saw you have a pr for NVDA Remote for this so I will look into that and apply the necessary changes for the unicorn addon.
Kind regards,
Falko

**Comment by LeonarddeR** [comment]
Hey falco,

You don't have to. See https://github.com/leonardder/unicornNVDA/tree/2023.1
Happy to pr that as soon as NVDA's API is stabilized for the new release.

**Comment by FalkoBabbage** [comment]
I have merged a pr based on your branch. Thanks @leonardder !
